test/extended/image_ecosystem/jenkins_plugin.go

```diff
@@ -153,8 +159,17 @@ var _ = g.Describe("[image_ecosystem][jenkins][Slow] openshift pipeline plugin",
 
 		g.By("kick off the build for the jenkins ephemeral and application templates")
 
+		var licensePrefix, pluginName string
 		newAppArgs := []string{exutil.FixturePath("..", "..", "examples", "jenkins", "jenkins-ephemeral-template.json")}
 		newAppArgs, useSnapshotImage := jenkins.SetupSnapshotImage(jenkins.UseLocalPluginSnapshotEnvVarName, localPluginSnapshotImage, localPluginSnapshotImageStream, newAppArgs, oc)
+		if !useSnapshotImage {
+			newAppArgs, useSnapshotImage = jenkins.SetupSnapshotImage(jenkins.UseLocalLoginPluginSnapshotEnvVarName, localLoginPluginSnapshotImage, localLoginPluginSnapshotImageStream, newAppArgs, oc)
+			licensePrefix = loginLicenseText
+			pluginName = loginPluginName
+		} else {
+			licensePrefix = originalLicenseText
+			pluginName = originalPluginName
+		}
```

Comment:
Sure I can rework along those lines.  Note there is a similar pattern in pipeline.go.  I'll rework that as well.